### PR TITLE
Bump DFP API client library version

### PR DESCRIPTION
**Pull request by kelvin-chappell**

### CODE REVIEW

File: admin/app/views/commercial/commercialMenu.scala.html [hunk 1]
```diff
@@ -19,7 +19,7 @@ <h3>Performance</h3>
                         <li><a href="@controllers.admin.routes.CommercialController.renderCommercialRadiator()">Commercial Radiator</a></li>
                         <li><a href="@controllers.admin.routes.CommercialController.renderBrowserPerformanceDashboard()">Real-user advert performance</a></li>
                         <li><a href="http://dashboards.gu-web.net/s3?list=dotcom-commercial/health/queries">Commercial Health</a></li>
-                        <li><a href="@controllers.admin.commercial.routes.TeamKPIController.renderPrebidDashboard()">Prebid vs Sonobi wrapper</a></li>
+                        <li><a href="@controllers.admin.commercial.routes.TeamKPIController.renderPrebidDashboard()">Prebid Bidder Performance</a></li>
```
Comment (JonNorman) [inline admin/app/views/commercial/commercialMenu.scala.html]: Is this commit intended to be in this PR? Given the innocuous nature I'm fine with you keeping it in here. It won't be the end of the world if this needs to be reverted.

**Review by JonNorman — APPROVED** [review]
Shall we have our annual conversation about whether we can factor out the version into a variable to make this job easier?

File: admin/app/views/commercial/commercialMenu.scala.html (hunk 1 above)
Comment (kelvin-chappell) [reply admin/app/views/commercial/commercialMenu.scala.html]: This is the seed for the PR!  I wanted to fix the link but I thought I'd better do something more useful at the same time.

Comment (JonNorman) [reply admin/app/views/commercial/commercialMenu.scala.html]: Ha I like it: justifying the work of the TeamCity agents.

**Comment by kelvin-chappell** [comment]
@JonNorman There's this project, which I've never got around to doing because we seem to be using the API in fewer places now than we used to: https://github.com/guardian/dfp-api-scala-wrapper

**Pull request merged**